extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -126,6 +126,12 @@ export const RENDER_IDLE_DELAY_REQUEST_EXP_BRANCHES = {
   EXPERIMENT: 21062232,
 };
 
+/**
+ * Map of pageview tokens to the instances they belong to.
+ * @private {!Object{string: !AmpAdNetworkDoubleclickImpl}
```

Comment:
this typing is wrong, believe it should be like `{!Object<string, !AmpAdNetworkDoubleclickImpl>}`

Reply:
Also not crazy about the naming, as it sounds like an array of pageViewStateTokens. Maybe like `tokensToInstances` toi make it somewhat clearer?

Reply:
Done.